packages/react-frontend/src/components/Comment.css

```diff
@@ -0,0 +1,39 @@
+.comment {
+    display: flex;
+    align-items: center;
+    background-color: #7c8a7a;
+    padding: 15px;
+    margin-bottom: 10px;
+    border-radius: 10px;
+    
+  }
+  
+  .comment-profile-circle {
+    width: 40px;
+    height: 40px;
+    background-color: #FFFF;
+    border-radius: 50%;
+    margin-right: 10px;
+    
+  }
+  
+  .comment-content {
+    color: white;
+    font-family: 'League Spartan', sans-serif;
+    font-style: normal;
+    font-size: 24;
+  }
+  
+  .comment-username {
+    font-weight: bold;
+    margin-bottom: 5px;
+    font-family: 'League Spartan', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+  }
+  
+  .comment-text {
+    color: white;
+    
+  }
```

Comment:
i like how you organized your css all of your component names are descriptive!

Reply:
i agree, very easy to read!